### PR TITLE
Added: ReactJS, NodeJS,PHP Khode Academy

### PR DESCRIPTION
**Pull request by vicheanath**
## What does this PR do?
Add resource(s)

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
Hmmm, the khmer list has some problems. The tracking parameters on the youtube likns should be removed, and each list should sorted alphabetically. (our linter is not activated for khmer.) just fix the sections you're adding to.

File: courses/free-courses-km.md [hunk 1]
```diff
@@ -29,10 +30,16 @@
 * [អនុវត្ត​កូដ Javascript](https://youtube.com/playlist?list=PLuEdNLfGOtnVmKfCI1gC6xHqJ_T9F85DW&si=cbbCi2zp_GoLcneK) - Khode Academy
 * [Javascript - បង្កើត​អន្តរកម្ម​គេហទំព័រ](https://youtube.com/playlist?list=PLuEdNLfGOtnUoeb8D2itGMIZayTi9ViOv&si=wxALcNb-i_4189KK) - Khode Academy
 * [ES6 - ជំនាន់​ថ្មី​របស់ Javascript](https://youtube.com/playlist?list=PLuEdNLfGOtnVOKm51qK8Gmx0tT-KbJoNd&si=-QXNo_NVAykbjPIt) - Khode Academy
+* [React.js - Web UI ជំនាន់​ថ្មី](https://youtube.com/playlist?list=PLuEdNLfGOtnVLr4irXpTsUiWtAq3PJHLy&si=VUszt37e9HanrZyU) - Khode Academy
+* [Node.js - Server-Side Javascript](https://youtube.com/playlist?list=PLuEdNLfGOtnW-wD7kT3rqZWrI_PlR3nsk&si=7XtDzjdviR42VyXJ) - Khode Academy
+
+
+### PHP
+
+* [PHP - កម្មវិធី​គេហទំព័រ](https://youtube.com/playlist?list=PLuEdNLfGOtnVsMxiXgZUuvqFKIavgZ-Bv&si=4p2obQA7R2-IxwMz) - Khode Academy
```
Comment (eshellman) [inline courses/free-courses-km.md:39]:
```suggestion
* [PHP - កម្មវិធី​គេហទំព័រ](https://youtube.com/playlist?list=PLuEdNLfGOtnVsMxiXgZUuvqFKIavgZ-Bv) - Khode Academy
```

File: courses/free-courses-km.md [hunk 2]
```diff
@@ -29,10 +30,16 @@
 * [អនុវត្ត​កូដ Javascript](https://youtube.com/playlist?list=PLuEdNLfGOtnVmKfCI1gC6xHqJ_T9F85DW&si=cbbCi2zp_GoLcneK) - Khode Academy
 * [Javascript - បង្កើត​អន្តរកម្ម​គេហទំព័រ](https://youtube.com/playlist?list=PLuEdNLfGOtnUoeb8D2itGMIZayTi9ViOv&si=wxALcNb-i_4189KK) - Khode Academy
 * [ES6 - ជំនាន់​ថ្មី​របស់ Javascript](https://youtube.com/playlist?list=PLuEdNLfGOtnVOKm51qK8Gmx0tT-KbJoNd&si=-QXNo_NVAykbjPIt) - Khode Academy
+* [React.js - Web UI ជំនាន់​ថ្មី](https://youtube.com/playlist?list=PLuEdNLfGOtnVLr4irXpTsUiWtAq3PJHLy&si=VUszt37e9HanrZyU) - Khode Academy
+* [Node.js - Server-Side Javascript](https://youtube.com/playlist?list=PLuEdNLfGOtnW-wD7kT3rqZWrI_PlR3nsk&si=7XtDzjdviR42VyXJ) - Khode Academy
```
Comment (eshellman) [inline courses/free-courses-km.md:34]:
```suggestion
* [Node.js - Server-Side Javascript](https://youtube.com/playlist?list=PLuEdNLfGOtnW-wD7kT3rqZWrI_PlR3nsk) - Khode Academy
* [React.js - Web UI ជំនាន់​ថ្មី](https://youtube.com/playlist?list=PLuEdNLfGOtnVLr4irXpTsUiWtAq3PJHLy) - Khode Academy

```

File: courses/free-courses-km.md [hunk 3]
```diff
@@ -29,10 +30,16 @@
 * [អនុវត្ត​កូដ Javascript](https://youtube.com/playlist?list=PLuEdNLfGOtnVmKfCI1gC6xHqJ_T9F85DW&si=cbbCi2zp_GoLcneK) - Khode Academy
 * [Javascript - បង្កើត​អន្តរកម្ម​គេហទំព័រ](https://youtube.com/playlist?list=PLuEdNLfGOtnUoeb8D2itGMIZayTi9ViOv&si=wxALcNb-i_4189KK) - Khode Academy
 * [ES6 - ជំនាន់​ថ្មី​របស់ Javascript](https://youtube.com/playlist?list=PLuEdNLfGOtnVOKm51qK8Gmx0tT-KbJoNd&si=-QXNo_NVAykbjPIt) - Khode Academy
+* [React.js - Web UI ជំនាន់​ថ្មី](https://youtube.com/playlist?list=PLuEdNLfGOtnVLr4irXpTsUiWtAq3PJHLy&si=VUszt37e9HanrZyU) - Khode Academy
```
Comment (eshellman) [inline courses/free-courses-km.md:33]:
```suggestion
```

**Review by eshellman — CHANGES_REQUESTED** [review]

File: courses/free-courses-km.md (hunk 2 above)
Comment (vicheanath) [reply courses/free-courses-km.md:34]: Okay I will change it.

**Pull request merged**